components/voice_registry_system.md

```diff
@@ -18,6 +18,9 @@ VRS's goal is to provide a consistent experience for the users and create an equ
 - **Query** - user’s word requesting for specific function and expecting a particular response.
 - **User** - a person who interacts with channels.  
 - **Voice Application** - also known as skill, action, capsule or domain.
+- **Voice assistant system** A system where a user (primarily) uses his/her voice to interact with an automated conversational assistant for information or to control devices. 
+The assistent responds to the user using verbal interaction, occasionally enhanced with visuals and visual interaction. The assistant system is controlled by an entity (any type of organisation).  Typical components are channel endpoints like a smart speaker or a mobile application. Wake word detection to activate the system. Automatic Speech Recognition to convert the users spoken words into written words. Natural Language Understanding to extract intents and meaning from these words. A back end to enable input and output, provide answers or control the devices. Dialog management to manage the conversation. Text to Speech and Voice generation to respond.
```

Comment:
Hi @Lens-fitzgerald  - I think we can eliminate these statements " Typical components are channel endpoints like a smart speaker or a mobile application. Wake word detection to activate the system. Automatic Speech Recognition to convert the user's spoken words into written words. Natural Language Understanding to extract intents and meaning from these words. A back end to enable input and output, provide answers or control the devices. Dialog management to manage the conversation. Text to Speech and Voice generation to respond."

I feel like the statement starts describing other things and not focus on the Voice Assistant System.

Reply:
Agree.